packages/gatsby-theme-docs/src/utils/helpers/compute-path-with-path-prefix.js

```diff
@@ -0,0 +1,8 @@
+import { withPrefix } from 'gatsby-link';
```

Comment:
```suggestion
import { withPrefix } from 'gatsby';
```

Reply:
I'm not sure why the suggestion if it works as is.